pages/community.page.js

```diff
@@ -22,6 +21,40 @@ function Community() {
       });
   }, []);
 
+  // Function to fetch commit count for each contributor
+  const fetchCommitCount = (login) => {
+    return axios
+      .get(`https://api.github.com/repos/anmode/grabtern-frontend/commits`, {
+        params: {
+          author: login,
+        },
```

Comment:
commit counts are not correct!

Reply:
I understand what you are try to say. I am working on fixing it. The issue is if no. of commit > 30 it is showing 30. I get it.